esmf_regrid/schemes.py

```diff
@@ -121,8 +125,15 @@ def _create_cube(data, src_cube, grid_dim_x, grid_dim_y, grid_x, grid_y):
     """
     new_cube = iris.cube.Cube(data)
 
-    new_cube.add_dim_coord(grid_x, grid_dim_x)
-    new_cube.add_dim_coord(grid_y, grid_dim_y)
+    if len(grid_x.points.shape) == 1:
+        new_cube.add_dim_coord(grid_x, grid_dim_x)
+    else:
+        new_cube.add_aux_coord(grid_x, (grid_dim_y, grid_dim_x))
```

Comment:
Is it sufficient to only add an AuxCoord or do we need to add a dim coord too?

Reply:
I don't think there is a DimCoord we could add here.